command.c

```diff
@@ -72,6 +75,7 @@ static int redis_arg0(struct cmd *r) {
 /*
  * Return true, if the redis command accepts exactly 1 argument, otherwise
  * return false
+ * Format: COMMAND key arg
```

Comment:
This comment is a bit confusing, since COMMAND here isn't the Redis command [COMMAND](https://redis.io/commands/command) but it can be any command.

Maybe lowercase `command` is better, as a placeholder just like `key` and `arg`.

Reply:
Ah, yes could be. Changed